cloudformation/panther-rules-engine-cross-account-read-only.yml

```diff
@@ -15,19 +15,19 @@ Parameters:
     AllowedPattern: '^\d{12}$'
   PantherRulesEngineRoleName:
     Type: String
-    Description: Panther rules-engine role name which can be found in the AWS Console under IAM by searching "rules" in the Panther Master Account
+    Description: Panther detections-engine role name which can be found in the AWS Console under IAM by searching "rules" in the Panther Master Account
```

Comment:
@wey-chiang could you review those changes?

Reply:
@georgesimos , we changed the name. This is valid.